api/endpoints/new-schedule-link.js

```diff
@@ -21,11 +21,15 @@ export default async (req, res) => {
     })
   }
 
+  // should open a meeting using
+  // <staging-app-url>/api/endpoints/schedule-link?id=9whz6
+
   // let's spice this name creation up in the future too
   const id = Math.random().toString(36).substring(7)
   res.json({id,
     videoUri: `https://hack.af/z-join?id=${id}`,
-    moreUri: `https://hack.af/z-phone?id=${id}`
+    moreUri: `https://hack.af/z-phone?id=${id}`,
+    stagingVideoUri: process.env.NODE_ENV === "staging" ? `https://slash-z-staging-1ae8b1c9e24a.herokuapp.com/api/endpoints/schedule-link?id=${id}` : null 
```

Comment:
I want to double check-- most staging apps on Heroku I've used are configured to change their subdomain (ie. `slash-z-staging-1ae8b1c9e24a`) after each commit. Will this URL stay consistent? If it will change, is there a way to not hard-code this?

Reply:
right now I manually deploy from `setup-staging` branch on Heroku and the URL has never changed since then so I thought it will be fixed